File: backend(spring)/iceback/src/main/java/com/ice/iceback/dto/response/Place.java
package com.ice.iceback.dto.response;

import java.util.Collection;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Place {
    private String place_name;
    private String road_address_name;
    private String address_name;
}

File: frontend(flutter)/lib/dto/Place.dart
class Place {
  final String placeName; //식당/카페 이름
  final String roadAddressName; // 도로 주소
  final String addressName; // 주소

  Place({required this.placeName, required this.roadAddressName, required this.addressName});

  factory Place.fromJson(Map<String, dynamic> json) {
    return Place(
      placeName: json['place_name'],
      roadAddressName: json['road_address_name'],
      addressName: json['address_name'],
    );
  }
}